hrl-examples/README.md

```diff
@@ -1 +1,141 @@
+5BD3E9B4.HRL: 0x4000 0x4fab f1ffffffffffff00b98500000000000064ea00791d24060000000000
+5BD3E9B4.HRL: 0x4000 0x4faf 7bffffffffffff023ba1284c77c5003032b6033ae1542f3df3134e6a
+5BD3E9B4.HRL: 0x4000 0x4fb3 0cffffffffffff01b955dbd7b6340019131e01738d79020000005046
+5BD3E9B4.HRL: 0x4000 0x4fb6 96ffffffffffff01395402b21038355fa08501bb7071fc07c95fea0a
+5BD3E9B4.HRL: 0x4000 0x4fba 24ffffffffffff007b7071fd37d11fe9251600905192650a00000080
+5BD3E9B4.HRL: 0x4000 0x4fbd aeffffffffffff007948a81d004002000000017a57695d7142000000
+5BD3E9B4.HRL: 0x4000 0x4fe9 8617550ce5ffff00b8527202383d200822a60087fe00000000200822
+5BD3E9B4.HRL: 0x4000 0x4fea 61ffffffffffff00395402b2101f1c0060b400903180580200fe1fff
+5BD3E9B4.HRL: 0x4000 0x4feb 41ffffffffffff00ba3ef4013a95fafc6f69003bc21055555500005a
+5BD3E9B4.HRL: 0x4000 0x4fec 23ffffffffffff009031bb5502000120ff3f007b7072023833200225
+5BD3E9B4.HRL: 0x4000 0x4fed 02ffffffffffff003ac201aa0d05aa0d0500001a2a00001916aa0d05
+5BD3E9B4.HRL: 0x4000 0x4fed daffffffffffff00155759631e00ff000001015031965102000c20ff
+5BD3E9B4.HRL: 0x4000 0x4fee b9ffffffffffff002a0c00f0452db00000000087fe00000000000000
+5BD3E9B4.HRL: 0x4000 0x4fef 98ffffffffffff007948970c004002000000003b399bf1fe00000180
+5BD3E9B4.HRL: 0x4000 0x4ff0 79ffffffffffff009051de640a000720ff3f01bb7072050892601d04
+5BD3E9B4.HRL: 0x4000 0x4ff1 56ffffffffffff007911b0fe36fdc90037f30187fe00000000da8e40
+5BD3E9B4.HRL: 0x4000 0x4ff2 30ffffffffffff007929d3263960f51fff3f004a40702e0000f51fff
+5BD3E9B4.HRL: 0x4000 0x4ff3 0fffffffffffff013f88ea1e00300000000001ba1e0000000000da7f
+5BD3E9B4.HRL: 0x4000 0x4ff3 f0ffffffffffff003a6301606e61507d15040187fe000000001e8c00
+5BD3E9B4.HRL: 0x4000 0x4ff4 ceffffffffffff017b7071fa37e31fff23390190310e5302000a1a2c
+5BD3E9B4.HRL: 0x4000 0x4ff5 a7ffffffffffff003b18120a21041b1d0da100bb18120a21041b1d0d
+5BD3E9B4.HRL: 0x4000 0x4ff6 82ffffffffffff0039327f89b7277b4fff0f00bc0184132800000000
+5BD3E9B4.HRL: 0x4000 0x4ff7 63ffffffffffff00905192650a000420ff3f003f2a00544731313831
+5BD3E9B4.HRL: 0x4000 0x4ff8 44ffffffffffff0039185b058518208000000079185b058518208000
+5BD3E9B4.HRL: 0x4000 0x4ff9 20ffffffffffff0079455d42002fa400020001bc0184132840000000
+5BD3E9B4.HRL: 0x4000 0x4ff9 f9ffffffffffff005b1a2b002068c900070200390322330000a0e020
+5BD3E9B4.HRL: 0x4000 0x4ffa d917550ce5ffff0087fe00000000fed01f000079556f7faf2d001583
+5BD3E9B4.HRL: 0x4000 0x4ffb b7ffffffffffff003b9afe90fffe90ff6c17003929592a0f600020ff
+5BD3E9B4.HRL: 0x4000 0x4ffc 99ffffffffffff00738dbc0d000000504600003ba2bd740000ec0080
+5BD3E9B4.HRL: 0x4000 0x4ffd 75ffffffffffff007918500a85181080000000939df95801009f0000
+5BD3E9B4.HRL: 0x4000 0x4ffe 4fffffffffffff01ba1e000000000022802001ba2eb0e9d3a74f9f3e
+5BD3E9B4.HRL: 0x4000 0x4fff 30ffffffffffff013b720a3102603c00f7000187fe0000000013fe69
+5BD3E9B4.HRL: 0x4000 0x5000 12ffffffffffff007975bb8938f526e0645101b945664b002fa40002
+5BD3E9B4.HRL: 0x4000 0x5000 f0ffffffffffff007948950a004002000000003ac10377d5f9460654
```

Comment:
I think it's three bytes that count as sequence number. Something like that:

magic number for block begin | sequence number  | unknown1 | unknown 2 (fixed?)  | payload
---------|---------- | --- | --- | ---
| 0x4000 | 3 bytes  | 0x1755_0ce5 OR 0xffff_ffff | 0xffff | remaining data |